### PR TITLE
Applied a number of changes to the debug menu

### PR DESCRIPTION
**Pull request by LOuroboros**
## Description
Story time.
Lunos needed a debug menu option to clear the bag yesterday.
The debug menu didn't have that, which made Lunos sad.
Lunos then thought about adding an option to clear the bag.
But Lunos didn't know where exactly should he place it inside the debug menu.
So Lunos made a lot of extra changes to make the option to clear the bag fit nicely in the debug menu teddiawwrsa
*Lunos could have just added the option in the debug menu's Utilities section, but Lunos is a masochist that enjoys to take the harder and longer paths to do things.*

So, to summarize, what did Lunos do in this PR?
-He renamed the debug menu's "Party/Boxes" to "Party".
-He renamed the debug menu's "Fill PC/Pockets" to "PC/Bag".
-He added the lovely Clear Bag feature to "PC/Bag".
-He moved "Access PC" and "Clear (Storage) Boxes" to "PC/Bag".
-He added a subcategory to PC/Bag to put the "Fill" options in for easier navigation.

## **Discord contact info**
lunos4026

### CODE REVIEW

**Comment by LOuroboros** [comment]
Also, Lunos fixed the alignment of the labels in `sDebugMenu_Items_Utilities` because why not.

<details>
  <summary>Click me</summary>
  He's joking, it's because that triggered him.
</details>

File: src/debug.c [hunk 1]
```diff
@@ -98,17 +98,35 @@ enum UtilMenu
...
 {
-    DEBUG_PARTY_BOXES_MENU_ITEM_ACCESS_PC,
-    DEBUG_PARTY_BOXES_MENU_ITEM_MOVE_REMINDER,
-    DEBUG_PARTY_BOXES_MENU_ITEM_HATCH_AN_EGG,
-    DEBUG_PARTY_BOXES_MENU_ITEM_HEAL_PARTY,
-    DEBUG_PARTY_BOXES_MENU_ITEM_POISON_MONS,
-    DEBUG_PARTY_BOXES_MENU_ITEM_CLEAR_BOXES,
+    DEBUG_PCBAG_MENU_ITEM_FILL,
+    DEBUG_PCBAG_MENU_ITEM_CLEAR_BAG,
+    DEBUG_PCBAG_MENU_ITEM_ACCESS_PC,
```
[the remark this answers is not in the document]
Comment (LOuroboros) [reply src/debug.c:105]: Putting it before the Fill subcategory kinda feels wrong, but I could totally put it above Clear Bag, sure.

[the remark this answers is not in the document]
Comment (AlexOn1ine) [reply src/debug.c:105]: Yes, I agree. Putting it as a second option sounds good.

[the remark this answers is not in the document]
Comment (LOuroboros) [reply src/debug.c:104]: Done :eyes:

mGBA_20240105_065800822

**Comment by AlexOn1ine** [comment]
If you want to return from the `Fill` back to the `PC/Bag` section, it goes back to the main menu. It's not big but it could be annoying at times. Could you change it if doesn't involve a lot of rewriting?

**Comment by LOuroboros** [comment]
> If you want to return from the `Fill` option back to the `PC/Bag` section, it goes back to the main menu. It's not big but it could be annoying at times. Could you change it if doesn't involve a lot of rewriting?

Sure thing, I'll look into it :eyes:

I already did something like this in the debug menu of my personal branch before the debug menu was heavily refactored, I'll look into what Lunos from the Past did and go from there.

**Comment by LOuroboros** [comment]
Done. Nice and easy.

mGBA_20240105_073342965

**Comment by Bassoonian** [comment]
Conflicts

**Comment by LOuroboros** [comment]
> Conflicts

Not anymore.

**Review by Bassoonian — APPROVED** [review]
